scripts/variantstore/wdl/GvsQuickstartIntegration.wdl

```diff
@@ -85,7 +85,6 @@ workflow GvsQuickstartIntegration {
                 vcf_files_column_name = wgs_vcf_files_column_name,
                 vcf_index_files_column_name = wgs_vcf_index_files_column_name,
                 sample_set_name = select_first([wgs_sample_set_name, "wgs_integration_sample_set"]),
-                use_classic_VQSR = false,
```

Comment:
why was this taken out?

Reply:
because of line 76 
It seemed like overkill to have both:
 use_VQSR_lite = true,
AND
 use_classic_VQSR = false,
 
but mainly because I was bullied into it by George and Miguel